Source/WebKit/UIProcess/Extensions/WebExtension.cpp

```diff
@@ -245,6 +248,67 @@ void WebExtension::populateWebAccessibleResourcesIfNeeded()
         parseWebAccessibleResourcesVersion2();
 }
 
+#if !PLATFORM(COCOA)
```

Comment:
I may remove this, as unless I can figure out and test a way (with WTF and WebCore) to expand a path string, it'll have to stay platform specific, as I'd prefer not to introduce vulnerabilities :)

Reply:
I looking into it, you can use `FileSystem::realPath()` to do this in a cross-platform way.

From the `man` page:
```
The realpath() function resolves all symbolic links, extra “/” characters, and references to /./ and /../ in file_name.
```